source/retryable-reads/tests/unified/handshakeError.yml

```diff
@@ -0,0 +1,76 @@
+description: "retryable reads handshake failures"
+
+schemaVersion: "1.3"
+
+runOnRequirements:
+  - minServerVersion: "4.2"
+    topologies: [replicaset, sharded, load-balanced]
+    auth: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent, connectionCheckOutStartedEvent]
+      uriOptions: { retryReads: true, retryWrites: true, maxPoolSize: 5 }
```

Comment:
Are `retryReads` and `retryWrites` relevant if they both already default to `true`?

Also, apologies if I asked this before but is there a significance to `maxPoolSize: 5`? Would 2 make more sense?